adler32_fold.h

```diff
@@ -0,0 +1,13 @@
+/* adler32_fold.h -- adler32 folding interface
+ * Copyright (C) 2022 Adam Stylinski
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#ifndef ADLER32_FOLD_H_
+#define ADLER32_FOLD_H_
+
+#include <stdint.h>
```

Comment:
Is this include necessary? It seems like whereever adler32_fold.h is included that stdint.h should already be included..

Reply:
It was mostly there for the uint32_t, but given how little is actually in this file I'm ok with foregoing it.  I mostly like to have the includes where needed there so that youcompleteme in vim doesn't get confused.  I'm sure visual studio or IDEs might also complain similarly.